### PR TITLE
Commom notificationBar use redux

### PR DESCRIPTION
**Pull request by TaimaTask**

### CODE REVIEW

File: src/pages/signUp/signupPageForm.jsx [hunk 1]
```diff
@@ -27,22 +25,41 @@ const SignupPageForm = () => {
 
   const errorCreateAccount = (error) => {
     dispatch(loadingActions.setIsLoading(false));
-    dispatch(errorMessageActions.errorMsg(error.message));
-    setTimeout(() => {
-      dispatch(errorMessageActions.errorMsg(''));
-    }, 1000);
+
```
Comment (IZablah) [inline src/pages/signUp/signupPageForm.jsx]: Remove line

File: src/pages/signUp/signupPageForm.jsx [hunk 2]
```diff
@@ -27,22 +25,41 @@ const SignupPageForm = () => {
...
+        position: "top",
+      })
+    );
   };
 
   const successCreateAccountInformation = () => {
+    dispatch(loadingActions.setIsLoading(false));
+    dispatch(
+      toastActions.toast({
+        isToast: true,
```
Comment (IZablah) [inline src/pages/signUp/signupPageForm.jsx]: What the idea for that?

File: src/pages/signIn/signinPageForm.jsx [hunk 3]
```diff
@@ -27,14 +25,27 @@ const SigninForm = () => {
...
-    }, 1000);
+    dispatch(
+      toastActions.toast({
+        isToast: true,
+        type: "failure",
+        message: error.message,
+        position: "top",
+      })
+    );
+
```
Comment (IZablah) [inline src/pages/signIn/signinPageForm.jsx]: Remove line

**Review by IZablah — CHANGES_REQUESTED** [review]

File: src/pages/signUp/signupPageForm.jsx (hunk 2 above)
Comment (TaimaTask) [reply src/pages/signUp/signupPageForm.jsx]: I remove it

**Review by IZablah — APPROVED** [review]

**Pull request merged**